docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.9"
```

Comment:
What does 3.9 as a version signify?

Reply:
This is an optional property, but good practice to write docker-compose version.
Should I remove this?

Reply:
Is it the `docker-compose` version or our file's/app's version?

Reply:
It is the `docker-compose` version.

Reply:
Let's keep it then.